### ISSUE TITLE
Error parsing xml files with files where ImportAction name="merge"

### ISSUE THREAD

**Issue by markearnest** [lispercat/sailpoint-iiq-dev-accelerator#23]
The following is a valid xml file but the plugin is reporting "error parsing" any time I open it. This appears to be happening with any merge files.
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE sailpoint PUBLIC "sailpoint.dtd" "sailpoint.dtd">
<sailpoint>
    <ImportAction name="merge">
        <Configuration name="SystemConfiguration">
            <Attributes>
                <Map>
                     some entries
                 </Map>
            </Attributes>
        </Configuration>
    </ImportAction>
</sailpoint>
```

**Comment by lispercat** [comment]
Hi Mark, I just released 1.0.33 a few minutes ago. Can you try to update
the plugin (usually happens automatically) and see if it fixes the issue?

**Comment by markearnest** [comment]
1.0.33 fixed it. closing
